docs/index.md

```diff
@@ -5,8 +5,10 @@ Epip Encounters is a UI and QoL mod targeting [Epic Encounters 2](https://docs.g
 ## Download
 
 !!! info "Language warning"
-    Only English is fully supported.
-    Visit [this sheet](https://docs.google.com/spreadsheets/d/1N590ol649CFsMJuU2-UNBqWQP8cONo-CnUCcC1TPRgI/edit#gid=479886788) to contribute to translations.
+    Only English is fully supported, however a translation effort is underway.  
+    Localization is only possible thanks to the members of our community. If you want to help with it - feel free to contribute at [our translation spreadsheet](https://docs.google.com/spreadsheets/d/1N590ol649CFsMJuU2-UNBqWQP8cONo-CnUCcC1TPRgI/).  
+    **Please** include some ways to contact you if you're contributing to the spreadsheet - your Discord username, your email - anything so that we can reach out with whatever questions we may have.  
+    If you want to contribute in a language that does not currently exist in the spreadsheet - contact the developers so we can add it.  
```

Comment:
As mentioned on Discord I think the message box takes up way too much space with this in - and it is all repeated stuff from the info sheet anyways